delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -54,30 +52,12 @@ def __init__(self, company):
         self.init_connection(company)
 
     def init_connection(self, company):
-        if company.postlogistics_test_mode:
```

Comment:
So this is not needed anymore with the new version of the api, I guess ?

Reply:
All the namespace/location tricks is included in the WSDL file, so I do not see the point to keep this here, as well as the verification of the TLS certificate which is working now.

Reply:
Good to know it has finally been fixed.